src/plugins/kibana_react/public/index.ts

```diff
@@ -31,3 +31,11 @@ export { Markdown, MarkdownSimple } from './markdown';
 export { reactToUiComponent, uiToReactComponent } from './adapters';
 export { useUrlTracker } from './use_url_tracker';
 export { toMountPoint } from './util';
+
+/** dummy plugin, we just want kibanaReact to have its own bundle */
+export function plugin() {
+  return new (class KibanaReactPlugin {
```

Comment:
Are we going to add proper tree shaking in `v7.8` to address the root problem?

Reply:
If we can get it working then yes